src/app/modules/shared/components/details-fields/details-fields.component.ts

```diff
@@ -270,7 +270,7 @@ export class DetailsFieldsComponent implements OnChanges, OnInit {
     const updatedDate = new Date(`${entryFormDate}T${updatedHour.trim()}`).toISOString();
     const initialDate = get(this.entryToEdit, date, updatedDate);
     const initialHour = formatDate(get(this.entryToEdit, date, updatedDate), 'HH:mm', 'en');
-    const dateHasNotChanged = updatedHour === initialHour;
+    const dateHasNotChanged = (initialDate === updatedDate) && (updatedHour === initialHour);
```

Comment:
Is the second condition necessary? For example, I deleted the second condition because it seems to me that it never gets to that point, and everything works correctly.

Reply:
@jase156